tests/acceptance/blocks/verticalresults.js

```diff
@@ -66,6 +69,19 @@ class VerticalResults {
     return this._selector.exists;
   }
 
+  /**
+   * wait for results to return based on search complete state 
+   * (default selector timeout of 10 seconds)
+   */
+   async waitOnSearchComplete () {
+    if(!this._preSearch.exists) {
+      await this._searchLoading();
+      await t.expect(this._searchLoading.exists).ok();
+    }
+    await this._searchComplete();
+    await t.expect(this._searchComplete.exists).ok();
```

Comment:
Does `await this._searchComplete()` being successful imply `t.expect(this._searchComplete.exists).ok()` automatically?

Reply:
test would still consider passed if `await this._searchComplete()` doesn't find any matching element